MuonAnalysis/MuonAssociators/interface/PropagateToMuonCore.h

```diff
@@ -26,19 +25,30 @@
 #include "TrackingTools/TrajectoryState/interface/TrajectoryStateOnSurface.h"
 #include "SimDataFormats/Track/interface/SimTrackContainer.h"
 #include "SimDataFormats/Vertex/interface/SimVertexContainer.h"
+#include "MuonAnalysis/MuonAssociators/interface/trackStateEnums.h"
 
-class DetLayer;
 class IdealMagneticFieldRecord;
 class TrackingComponentsRecord;
 class MuonRecoGeometryRecord;
 
-class PropagateToMuon {
-public:
-  explicit PropagateToMuon(const edm::ParameterSet &iConfig, edm::ConsumesCollector);
-  ~PropagateToMuon();
+//This is a nearly identical copy to MuonAnalysis/MuonAssociators/interface/PropagateToMuonCore.h
```

Comment:
I'm not sure I understand this comment. Isn't this file the same that contains the comment?

Reply:
There is a nearly identical copy of the PropagateToMuon class, for reasons I don't know, in HLTrigger/Offline, and I somewhat randomly used that one as the starting point for the refactoring, hence the leftover comment. I removed it.